samples/peripheral/radio_test/src/radio_test.h

```diff
@@ -186,4 +189,11 @@ void radio_rx_stats_get(struct radio_rx_stats *rx_stats);
  */
 void toggle_dcdc_state(uint8_t dcdc_state);
 
+/**
+ * @brief Function for setting the nRF21540 activation delay.
+ *
+ * @param[in] delay Activation delay in microseconds.
+ */
+void nrf21540_active_delay_set(uint32_t delay);
```

Comment:
Prefix it with radio_ or radio_test. This looks like API from nrf21540.h

Reply:
This function is removed here and moved to nrf21540.h